API Testing/BTN Testing/environments/local.bru

```diff
@@ -0,0 +1,2 @@
+vars {
```

Comment:
I think we need to initilize the bearerToken for this to work stright off the bat
```suggestion
vars {
    bearerToken: ""
}
```